2019-05-20 09:10:57,083 ERROR: Exception on /register [POST] [in /Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/sabinapringle/Desktop/projects/ohapp/app/routes.py", line 85, in register
    return redirect(url_for('Login to interview'))
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 356, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2061, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 345, in url_for
    force_external=external)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/werkzeug/routing.py", line 2181, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'Login to interview'. Did you mean 'register' instead?
2019-05-20 09:13:29,242 ERROR: Exception on /register [POST] [in /Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/sabinapringle/Desktop/projects/ohapp/app/routes.py", line 85, in register
    return redirect(url_for('Login to interview'))
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 356, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2061, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 345, in url_for
    force_external=external)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/werkzeug/routing.py", line 2181, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'Login to interview'. Did you mean 'register' instead?
2019-05-20 14:24:34,957 INFO: Ohapp startup [in /Users/sabinapringle/Desktop/projects/ohapp/app/__init__.py:51]
2019-05-20 14:26:50,653 ERROR: Exception on /register [POST] [in /Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/sabinapringle/Desktop/projects/ohapp/app/routes.py", line 85, in register
    return redirect(url_for('Login to interview'))
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 356, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2061, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 345, in url_for
    force_external=external)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/werkzeug/routing.py", line 2181, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'Login to interview'. Did you mean 'register' instead?
2019-05-20 14:31:30,318 INFO: Ohapp startup [in /Users/sabinapringle/Desktop/projects/ohapp/app/__init__.py:51]
2019-05-20 14:32:44,842 ERROR: Exception on /register [POST] [in /Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/sabinapringle/Desktop/projects/ohapp/app/routes.py", line 85, in register
    return redirect(url_for('Login to interview'))
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 356, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/app.py", line 2061, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/flask/helpers.py", line 345, in url_for
    force_external=external)
  File "/Users/sabinapringle/Desktop/projects/ohapp/venv/lib/python3.7/site-packages/werkzeug/routing.py", line 2181, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'Login to interview'. Did you mean 'register' instead?
2019-05-20 14:57:23,170 INFO: Ohapp startup [in /Users/sabinapringle/Desktop/projects/ohapp/app/__init__.py:51]
